N2-55 Migrate Inflection fields.

jonholdsworth opened the pull request:
Inflection fields appear as Boolean choices under 'Morphology' collapsible on Change Gloss page.
This PR needs to be merged simultaneously with [PR _N2-55 Migrate Inflection fields_ on **nzsl-freelex-signbank-data-migration** repository](https://github.com/ackama/nzsl-freelex-signbank-data-migration/pull/25).
![image](https://user-images.githubusercontent.com/82071930/170913897-91fe2fbe-0e0b-48a4-946f-74168ff1e04e.png)

jonholdsworth committed:
N2-55 Migrate Inflection fields.
Inflection fields appear as Boolean choices under 'Morphology' collapsible on Change Gloss page.

joshmcarthur commented on signbank/dictionary/admin.py:
```diff
@@ -183,6 +183,8 @@ class GlossAdmin(VersionAdmin):
     fieldsets = ((None, {'fields': ('dataset', 'published', 'exclude_from_ecv', 'id', 'idgloss', 'idgloss_mi', 'notes', 'hint', 'signer', 'filmbatch')},),
                  (_('Created/Updated'), {'fields': ('created_at',
                   'created_by', 'updated_at', 'updated_by')},),
+                 (_('Morphology'), {'fields': ('inflection1', 'inflection2', 'inflection3'),
```
These column names are not descriptive, and it sounds like they can be, since they have descriptive labels in the model definition.

I would suggest:

* `inflection_temporal`
* `inflection_manner_degree`
* `inflection_plural`

jonholdsworth replied:
Sure thing.

joshmcarthur commented:
Also noting that I can't see in this patch where the morphology fields appear in the advanced gloss page. This seems to only add them to the admin

jonholdsworth commented:
> Also noting that I can't see in this patch where the morphology fields appear in the advanced gloss page. This seems to only add them to the admin

Correct. As per discussion we need to investigate how to add them to the advanced gloss page. That will be a near-future update and a new PR.

jonholdsworth committed:
N2-55 Migrate inflection fields.
In Signbank code these have been renamed from inflection1 inflection2 inflection 3
to inflection_temporal inflection_manner_degree inflection_plural respectively, for improved readability.
